packages/typespec-python/test/azure/generated/resiliency-srv-driven2/resiliency/srv/driven2/_client.py

```diff
@@ -25,24 +25,23 @@ class ResiliencyServiceDrivenClient(ResiliencyServiceDrivenClientOperationsMixin
 
     There are three concepts that should be clarified:
 
-
-    #. Client spec version: refers to the spec that the client is generated from. 'v1' is a client
+    1. Client spec version: refers to the spec that the client is generated from. 'v1' is a client
     generated from old.tsp and 'v2' is a client generated from main.tsp.
-    #. Service deployment version: refers to a deployment version of the service. 'v1' represents
+    2. Service deployment version: refers to a deployment version of the service. 'v1' represents
     the initial deployment of the service with a single api version. 'v2' represents the new
     deployment of a service with multiple api versions
-    #. Api version: The initial deployment of the service only supports api version 'v1'. The new
+    3. Api version: The initial deployment of the service only supports api version 'v1'. The new
```

Comment:
After replace m2r, there is some diff of docstring in generated SDK, are these changes expected? @catalinaperalta

Reply:
Yes this is an improvement compared with m2r. Based on my investigation when you want to have a list that's automatically ordered for you specify it with hashtags. I've looked at the diff and I think we're doing the right thing overall